### DISCUSSION TITLE
How to Implement table Infinite Pagination ?

### DISCUSSION THREAD

**Discussion by theqewk** [filamentphp/filament#7678] [Help]
How to build a table with infinite pagination to dynamically fetch data from the back end using cursor pagination for faster and more efficient query execution?
So that when the table is scrolled, the data is loaded to the bottom of the table.

**Comment by pboivin** [comment]
I don't think this is possible with the Filament Table. It may be easier to just build your own Livewire table and reuse some of the HTML and CSS from Filament.
